Weather/FlagoftheCharles.15m.rb

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/ruby
```

Comment:
Can you make sure this file is executable - `chmod +x Weather/FlagoftheCharles.15m.rb; git add Weather/FlagoftheCharles.15m.rb; git commit -m'make FlagoftheCharles.15m.rb executable'` should do the trick 😄